### PR TITLE
Change formatting of News

### PR DESCRIPTION
**Pull request by three**
This format supports multiple stories and will display the dates each was posted. I've added the Join Rocketry post (57ef40e61a0fd4b1a985d6fee9f9aa65866df8ed) back to demonstrate.

### CODE REVIEW

File: news.html [hunk 1]
```diff
@@ -22,13 +22,25 @@
         </div>
 
         <h1>NEWS</h1>
-        <div class="enclosed-box infobox">
-            <h2>Club Meeting Friday 1/8</h2>
+        <!-- Put news in reverse chronological order. -->
+
+        <div class="post infobox">
+            <h1 class="post-title">Club Meeting Friday 1/8</h1>
```
Comment (winstliu) [inline news.html]: Are you sure these deserve to be `h1`?

File: style.css [hunk 2]
```diff
@@ -67,3 +67,14 @@ p {
...
+
+/*
+ * News
+ */
+
+.post {
+    margin-bottom: 12px;
+}
+.post-title {
+    font-size: 30px;
```
Comment (winstliu) [inline style.css]: I would really like if we didn't add any extra font sizes.  Linters start complaining once you add too many.  `h2` is already 30px.

File: style.css [hunk 3]
```diff
@@ -67,3 +67,14 @@ p {
...
 }
+
+/*
+ * News
+ */
+
+.post {
+    margin-bottom: 12px;
+}
+.post-title {
```
Comment (winstliu) [inline style.css]: Newline separating these two?

File: news.html (hunk 1 above)
Comment (three) [inline news.html]: I based it on the format used by the [Bootstrap blog](http://blog.getbootstrap.com/).

**Comment by winstliu** [comment]
Looks good to me.

**Pull request merged**